test/tools/jsontests/TransactionTests.cpp

```diff
@@ -79,8 +79,9 @@ json_spirit::mObject FillTransactionTest(json_spirit::mObject const& _o)
 	mValue expectObj = _o.at("expect");
 	for (auto const network: test::getNetworks())
 	{
-		unique_ptr<SealEngineFace> se(ChainParams(genesisInfo(network)).createSealEngine());
-		bool onConstantinople = (network == eth::Network::ConstantinopleTest);
+		ChainParams params(genesisInfo(network));
+		unique_ptr<SealEngineFace> se(params.createSealEngine());
+		bool onConstantinople = (params.constantinopleForkBlock == 0); //Test networks has forkblocks set to 0 if rules are enabled
```

Comment:
Why is this zero for true?

Reply:
Test networks has forkblocks set to 0 if rules are enabled. because it indicates the blocknumber from which it start to work

Reply:
transaction tests does not have blocknumber anymore

Reply:
Ah, OK.  The current block number is 1, so let's compare it against 1.

If ConstantinopleForkBlock = 1, we should be already on Constantinople.

Reply:
it is set to 0. do you mean 
` if 1 >= ConstantinopleForkBlock  ` ?

Reply:
Yeah, the theoretical block number is 1, not zero.